pkg/sql/sem/tree/datum.go

```diff
@@ -4434,11 +4434,11 @@ func (d *DArray) Format(ctx *FmtCtx) {
 	}
 
 	ctx.WriteString("ARRAY[")
-	comma := ""
+	delimiter := ""
 	for _, v := range d.Array {
-		ctx.WriteString(comma)
+		ctx.WriteString(delimiter)
 		ctx.FormatNode(v)
-		comma = ","
+		delimiter = d.ParamTyp.Delimiter()
```

Comment:
The error is here - we should undo this. I believe the change only affects the {} syntax.

Reply:
I will drop that commit.

Reply:
For posterity: I removed the changes to line 4441:

```golang
delimiter = d.ParamTyp.Delimiter()
```

Reply:
hmm, this still is here and causing the tests to fail. i think all of this change should be removed.

Reply:
My bad I typoed a Git command to drop the commit.